mmil.raw.html

```diff
@@ -727,115 +727,85 @@
+<LI>Definition of proper substitution ~ df-sb</LI>
 
-<LI>
-<A HREF="2eu7.html">Double existential uniqueness</A></LI>
+<LI>Double existential uniqueness ~ 2eu7</LI>
 
 </MENU>
 <B>Set theory</B>
 <MENU>
-<LI>
-<A HREF="uncom.html">Commutative law for union</A></LI>
+<LI>Commutative law for union ~ uncom</LI>
 
-<LI>
-<A HREF="abeq2.html">A basic relationship between class and wff
-variables</A></LI>
+<LI>A basic relationship between class and wff
+variables ~ abeq2</LI>
 
-<LI>
-<A HREF="isset.html">Two ways of saying &quot;is a set&quot;</A></LI>
+<LI>Two ways of saying &quot;is a set&quot; ~ isset</LI>
 
-<LI>
-<A HREF="regexmid.html">The ZF axiom of foundation implies excluded middle</A></LI>
+<LI>The ZF axiom of foundation implies excluded middle ~ regexmid</LI>
 
-<LI>
-<A HREF="ru.html">Russell's paradox</A></LI>
+<LI>Russell's paradox ~ ru</LI>
 
-<LI>
-<A HREF="ordtriexmid.html">Ordinal trichotomy implies excluded middle</A></LI>
+<LI>Ordinal trichotomy implies excluded middle ~ ordtriexmid</LI>
 
-<LI>
-<A HREF="findes.html">Mathematical (finite) induction</A></LI>
+<LI>Mathematical (finite) induction ~ findes</LI>
 
 <LI>Peano's postulates for arithmetic:
-<A HREF="peano1.html">1</A>
-<A HREF="peano2.html">2</A>
-<A HREF="peano3.html">3</A>
-<A HREF="peano4.html">4</A>
-<A HREF="peano5.html">5</A></LI>
+~ peano1 ~ peano2 ~ peano3 ~ peano4 ~ peano5</LI>
 
-<LI><A HREF="nndceq.html">Two natural numbers are either equal or not equal</A> (a special case of the law of the excluded middle which we can prove).</LI>
+<LI>Two natural numbers are either equal or not equal ~ nndceq (a special case of the law of the excluded middle which we can prove).</LI>
 
-<LI><A HREF="nn0suc.html">A natural number is either zero or a successor</A></LI>
+<LI>A natural number is either zero or a successor ~ nn0suc</LI>
 
-<LI>
-<A HREF="acexmid.html">The axiom of choice implies excluded middle</A></LI>
+<LI>The axiom of choice implies excluded middle ~ acexmid</LI>
 
-<LI>
-<A HREF="onprc.html">Burali-Forti paradox</A></LI>
+<LI>Burali-Forti paradox ~ onprc</LI>
 
-<LI>
-<A HREF="tfis3.html">Transfinite induction</A></LI>
+<LI>Transfinite induction ~ tfis3</LI>
 
-<LI>
-<A HREF="oacl.html">Closure law for ordinal addition</A></LI>
+<LI>Closure law for ordinal addition ~ oacl</LI>
 
 </MENU>
 <B>Real and complex numbers</B>
 
 <MENU>
-<LI> <A HREF="arch.html">Archimedean property of real numbers</A></LI>
+<LI>Archimedean property of real numbers ~ arch</LI>
 
 <LI>Properties of apartness:
-<A HREF="apirr.html">1</A>
-<A HREF="apsym.html">2</A>
-<A HREF="apcotr.html">3</A>
-<A HREF="apti.html">4</A></LI>
+~ apirr ~ apsym ~ apcotr ~ apti</LI>
 
-<LI> <A HREF="sqrt2irrap.html">The square root of 2 is irrational</A> (a
-different statement than <A HREF="sqrt2irr.html">The square root of 2
-is not rational</A>)</LI>
+<LI>The square root of 2 is irrational ~ sqrt2irrap (a
+different statement than The square root of 2
+is not rational ~ sqrt2irr )</LI>
```

Comment:
~ sqrt2irrap does not exist, I think this must be ~ sqrt2irr0.

Maybe ~sqrt2irr should be renamed to "sqrt2nq", then ~sqrt2irr0 can be renamed to "sqrt2irr"

Furthermore, I would propose to add quotation marks:
(a different statement than "The square root of 2 is not rational" ~ sqrt2irr )

Reply:
Sorry, this is the theorem sampler for iset! Here we actually have ~sqrt2irrap: ``( Q e. QQ -> ( sqrt ` 2 ) =//= Q )``.
But this is in the same way not precisely the meaning of "is irrational": `A e. ( RR \ QQ )` is the adequate idiom to say "`A` is irrational".

Reply:
By the way, why don't we have ``( sqrt ` 2 ) e. ( RR \ QQ )`` in iset.mm?

Reply:
I've added the quotes, and added `sqrt2irr0` to iset.mm.

But the usual definition of irrational, in the absence of excluded middle, is `sqrt2irrap` rather than `sqrt2irr`. This is discussed in the comments for those theorems and if you want an external site which discusses it, see https://en.wikipedia.org/wiki/Irrational_number#In_constructive_mathematics or https://ncatlab.org/nlab/show/irrational+number

Or if you want an example of where this would matter, look at the antecedent of https://us.metamath.org/ileuni/flapcl.html

The reason I like having this in the iset.mm theorem sampler is, of course, because this may be a new concept to a lot of readers.